keps/prod-readiness/sig-network/1860.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group 
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 1860
+alpha:
+  approver: "@johnbelamaric"
```

Comment:
Feel free to reassing to me - and I will take a look in the next 2 weeks (I'm afraid it may take John visibly more).